kuttl_tests.md

```diff
@@ -0,0 +1,69 @@
+Writing KUTTL test for an operator
+==================================
+
+Kuttl is a tool for testing Kubernetes operators. Each kuttl test suite
+consists of a series of test steps that might modify the state of the cluster
+or check its state. A kuttl test step will use all yaml files that begin with
+its numeric index. For example, if we had a file 00-deploy.yaml and
+a 00-assert.yaml, kuttl would apply the changes described by the file
+00-deploy.yaml and then wait until the state of the cluster matches the one
+found in 00-assert.yaml. If it's found, then the step is successful. If after
+the timeout, the state of the cluster does not match the assert, then the test
+fails. Kuttl also supports errors files, which will check that the cluster does
+not match the state described in the file.
```

Comment:
(i.e. the inverse of 'assert') 

just suggestion

Reply:
Added.